recipe/meta.yaml

```
@@ -14,6 +14,7 @@ source:
 
 build:
   number: 0
+  trigger: 0
```

```suggestion
  # trigger 1
```
It shouldn't be a key but a comment